AGENTS.md

```
@@ -14,23 +24,36 @@ English is mandatory for all project-facing work and communication.
 - When quoting or preserving non-English input, add an English explanation and keep the non-English text as short as possible.
 - If a contributor opens an issue, comment, or review in another language, respond in English and continue the thread in English.
 
-## Architecture
+## Repository Structure
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)